autocannon.js

```
@@ -5,6 +5,7 @@
 const minimist = require('minimist')
 const fs = require('fs')
 const path = require('path')
+const URL = require('url').URL
```

Can you throw an error if `URL` is not available?